src/components/ActionPage/Hero.tsx

```diff
@@ -76,8 +80,37 @@ const SubtitleContainer = styled.div`
   }
 `;
 
+const NewLine = () => <br />;
+
 export default function Hero() {
-  const { active } = useWeb3();
+  const { active, account, chainId, library } = useWeb3();
+  const backoffice = useMiniBackoffice(account, chainId, library);
+  const [registryStatus, setRegistryStatus] = useState<string>("");
+
+  useEffect(() => {
+    let status = "";
+    switch (backoffice.status) {
+      case "UnregisteredUser":
+        status =
+          " but your address is not in the DID Registry. So, you can't purchase yet! Use the Backoffice control in step #2 to update the DID Registry.";
+        break;
+      case "KYCAbsent":
+        status =
+          " and your address is in the DID Registry but it is not in the KYC List. So, you can't purchase yet! Use the Backoffice control in step #2 to update the DID Registry.";
+        break;
+      case "KYCApproved":
+        status =
+          ", your address is in the DID Registry AND you are in the KYC List. So, you can purchase! ";
+        break;
+      case "Loading":
+        status = " but your status is getting updated in the Registry.";
+        break;
+
+      default:
```

Comment:
Let's make TS complain if we forget to cover any `backoffice.status` possibility.
```suggestion
      default:
        unreachable(backoffice);
```